packages/core/src/description-list/description-list.scss

```diff
@@ -0,0 +1,24 @@
+@import './../styles/tokens/generated/index';
+
+dl[cds-list] {
+  dt {
+    font-weight: bold;
+  }
+
+  dd {
+    margin-left: 0;
+  }
+}
+
+@mixin renderGapStyles($sizes: null) {
+  @each $size in $sizes {
+    dl[cds-list][cds-layout~='vertical'][cds-layout*='gap:#{$size}'] {
+      gap: 0;
```

Comment:
if you try and wrap your `dd` and `dt` in a div like the horizontal example and use a `vertical` layout with spacing, this will break unfortunately.

Reply:
I suppose I could remove all of the complexity and say "if you want a gap between items, wrap in a div" and then `cds-layout` would work out of the  box

Reply:
Let's recommend wrapping the `dd` and `dt` in a div for vertical layouts as well. This will keep the layout behavior consistent when using `cds-layout`. Also ensures the with theming/responsive the gap behavior is the same. The vertical layouts seem to work as long as they are wrapped. https://stackblitz.com/edit/js-1qackg?file=index.html